### PR TITLE
PyTorch Autolog Example Readme files 

### PR DESCRIPTION
**Pull request by karthik-77**
…cation and Mnist example file

Signed-off-by: karthik-77 <karthiks@ideas2it.com>

## What changes are proposed in this pull request?

Updated readme files for the pytorch autolog examples :bert-news-classification,mnist -example1,example2

## How is this patch tested?

Documentation - NA

### Is this a user-facing change?

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [X] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/server-infra`: MLflow server, JavaScript dev server
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: examples/pytorch/BertNewsClassification/README.md [hunk 1]
```diff
@@ -1,22 +1,5 @@
...
-This is an experimental api that logs ML model artifacts and metrics.
-The metrics are logged during training of the model.
-
-## Setting tracking URI
-MLflow tracking URI can be set using the environment variable MLFLOW_TRACKING_URI
-
-Example: `export MLFLOW_TRACKING_URI=http://localhost:5000/`
-
-For more details - https://mlflow.org/docs/latest/tracking.html#where-runs-are-recorded
+## BERT news classification example
```
Comment (smurching) [inline examples/pytorch/BertNewsClassification/README.md:1]: @karthik-77 I reworked this file with a couple of changes:
* Simplified the intro & removed the "code related to MLflow" section, which is redundant given the description of ``mlflow.pytorch.autolog`` here
* Split out the ``mlflow ui`` command into its own "Viewing results in the UI" section and moved it above the description of how to customize training parameters (to streamline the common path of running the example with default parameters -> viewing results in the UI).
* Reworded the description of training parameters 

Would you be able to apply similar changes to the other READMEs? Thanks!

**Comment by karthik-77** [comment]
Hi Siddarth,

Sure i will rework on the same lines, and make the changes accordingly in
the respective files.

Thanks and Regards
karthik

File: examples/pytorch/BertNewsClassification/README.md [hunk 2]
```diff
@@ -41,35 +24,45 @@ If you have the required modules for the file and would like to skip the creatio
...
-1. Number of epochs - max_epochs
-2. Number of gpus - gpus
-3. Backend in case of gpus environment - accelerator
-4. Batch size to process - batch-size
-5. Number of workers to process input - num-workers
-6. Learning rate - lr
 
 For example:
 ```
+mlflow run . -P epochs=5 -P gpus=1 -P batch_size=32 -P num_workers=2 -P learning_rate=0.01 -P accelerator="ddp"
```
Comment (smurching) [inline examples/pytorch/BertNewsClassification/README.md:53]: @karthik-77 I added an example here of how to run the training script as an MLflow project with custom parameters, for consistency / logical flow from the previous section, which explains how to run training script as a project. Could we add a similar section to the other READMEs as well?

**Review by smurching — COMMENTED** [review]
Thanks @karthik-77 for the PR! I noticed a few other issues with the BERT example that it'd be great to address in another PR:

* We have a parameter called ``max_epochs`` in the script, but ``epochs`` in the MLproject file. Could we make the two consistent (e.g. ``max_epochs`` in both places?)
* There’s a ``num_workers`` param documented to have a default of 0 [here](https://github.com/mlflow/mlflow/blob/055978cdd425ac7c328069d80230f5bc50265df8/examples/pytorch/BertNewsClassification/bert_classification.py#L172), but the default is really 3. Could we update the linked line with the correct default value?

**Comment by karthik-77** [comment]
Hi Siddarth,

Sure will make those changes so that consistency is maintained throughout.
Thanks for pointing that out.

Thanks and Regards
karthik

**Comment by karthik-77** [comment]
@smurching  Request you to kindly sign off the commits.

File: examples/pytorch/BertNewsClassification/README.md (hunk 1 above)
Comment (karthik-77) [reply examples/pytorch/BertNewsClassification/README.md:1]: I have applied similar changes in the other read me files (Mnist-example1,example2) and updated the PR

**Review by smurching — APPROVED** [review]
LGTM

**Pull request merged**